app/controllers/ActivateAccountController.scala

```diff
@@ -37,4 +40,21 @@ class ActivateAccountController @Inject() (
         Future.successful(Ok("issues with activation link"))
     }
   }
+
+  private def createConnectionIfSharedLink(redirect: String, user: User): Unit = {
```

Comment:
Curious how this type-checks since it actually returns `Option` when condition is true right?

Reply:
The return type is Unit so returned value is discarded I believe